src/scss/components/checkbox/_config.scss

```diff
@@ -0,0 +1,8 @@
+/// ========== VARIABLES ========== ///
+@import "vars";
+
+/// ========== MIXINS ========== ///
+@import "components/form/config";
+@import "./mixins/hxCheckboxControl";
+@import "./mixins/hxCheckboxFacade";
+@import "mixins";
```

Comment:
The global mixins may not need imported for this component.

I'm not seeing any use of a mixin defined in `src/scss/_mixins.scss`.

Reply:
Removed `@import "mixins";`